doc/guides/coverage.rst

```diff
@@ -141,6 +141,13 @@ or::
 
 which will produce ``sanity-out/coverage/index.html`` with the report.
 
+Note that the sanitycheck coverage reports are currently created with
+`lcov`_, with a minimum version of 1.14. Some Linux distributions may
+not be up-to-date.
```

Comment:
Can you provide instructions for how to get a more recent version of lcov.  I tried a ``sudo apt install lcov`` on my Ubuntu system and it got version 1.13-3  :(

Reply:
I do not think we should go into this level of details, every distro is different, whatever we will say here will likely not work for most people, so leave this as an exercise for the user to figure this out.

Reply:
That wouldn't be a good "Developer Experience" (just let them figure it out on their own).  Can we at least point to some installation site for lcov, (maybe https://github.com/linux-test-project/lcov/blob/master/README ) where it documents how to get the source and build it?